### PR TITLE
Addition of 'bulletPosition' prop to help position the bullets.

### PR DESCRIPTION
**Pull request by rishabh-saxena**
A prop 'bulletPosition' can be passed into the Carousel to position the bullets in the carousel.

### CODE REVIEW

**Comment by phil-r** [comment]
This should be called `bulletsStyle`

File: index.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ export default class Carousel extends Component {
     arrows: PropTypes.bool,
     arrowsContainerStyle: Text.propTypes.style,
     arrowStyle: Text.propTypes.style,
+    bulletPosition: Text.propTypes.style,
```
Comment (phil-r) [inline index.js]: Imho `bulletsStyle` is a better name for this

**Comment by CodyBrouwers** [comment]
Would like to see this merged if possible, not sure what the best naming convention for the prop is but it is definitely valuable functionality.

**Comment by phil-r** [comment]
@CodyBrouwers feel free to use `bulletsContainerStyle` but in most cases you'll need to overwrite defaults: https://github.com/phil-r/react-native-looped-carousel/blob/master/index.js#L430

**Comment by CodyBrouwers** [comment]
Great, thanks @phil-r!

**Pull request merged**